src/Shared/FrameworkLocationHelper.cs

```diff
@@ -327,6 +333,7 @@ internal static class FrameworkLocationHelper
                 dotNetFrameworkVersion471,
                 dotNetFrameworkVersion472,
                 dotNetFrameworkVersion48,
+                dotNetFrameworkVersion481,
```

Comment:
This is the only one I expected, but to be super clear I'm not 100% confident I understand the current functionality of these VisualStudioSpecs